classes/web_processor.php

```diff
@@ -66,6 +66,7 @@ function () use ($manager) {
                 $manager->get_timer()->stop();
                 $manager->get_profiler()->stop();
                 $this->process($manager, true);
+                page_group::record_fuzzy_counts($this->profile);
```

Comment:
this is called in process above, do we need this here?

Reply:
record_fuzzy_counts() is called in one place in web_processor, and in one place in cron_processor. These classes are not active at the same time, so record_fuzzy_counts() should be called only once per run.

Reply:
ah sorry yes, so to be consistent should this web one be inside process?

Reply:
Oops, I was mistaken a little there. Let's see if I can explain it better.

`web_processor` creates one profile for the run. `process()` is called multiple times for the same profile in order to make partial saves. Fuzzy counting is done after the final `process()` call. 

`cron_processor` creates a profile for each task it encounters. `process()` is called once for each profile. So fuzzy counting is done each time.

Either way, we do one fuzzy count for each profile attempt. Is this correct?

I moved the call to fuzzy counting out of `cron_processor::process()` anyway.

Reply:
all good